module-code/app/securesocial/core/providers/ChatworkProvider.scala

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2015 Mikael Vallerie
```

Comment:
Remove this, or change it to your name.

Reply:
Instead of Vallerrie-san, I write my name in it.